lib/config/plugins.js

```diff
@@ -114,6 +114,16 @@ module.exports = {
             longName = pluginNamespace + PLUGIN_NAME_PREFIX + pluginNameWithoutPrefix;
         let plugin = null;
 
+        if (pluginName.match(/\s+/)) {
+            const whitespaceError = new Error("Whitespace found in plugin name eslint-plugin-" + pluginNameWithoutPrefix);
+
+            whitespaceError.messageTemplate = "whitespace-found";
+            whitespaceError.messageData = {
+                pluginName: pluginNameWithoutPrefix
```

Comment:
Could you use `longName` in error messages? Package names might have scope qualifiers (e.g. `@mysticatea/eslint-plugin-test`)
Related to #6939